src/System.Threading.Overlapped/tests/System.Threading.Overlapped.Tests.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+    <AssemblyName>System.Threading.Overlapped.Tests</AssemblyName>
+    <ProjectGuid>{861A3318-35AD-46ac-8257-8D5D2479BAD9}</ProjectGuid>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
This project should be marked as not being usable on Linux or OSX, e.g. https://github.com/dotnet/corefx/blob/master/src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj#L11